### PR TITLE
feat(ui-compo): migrate topBar component + fix redirect error

### PR DESCRIPTION
**Pull request by kengork**
# PR Prelude

Thank you for contributing to sheikah! :)

**Please fill in this template (by putting an `x` inside
the brackets) _before_ filing your PR:**

- [x] I have read and understood [CODE_OF_CONDUCT][code] document.
- [x] I have read and understood [CONTRIBUTING][cont] document.
- [x] I have read and understood [STYLEGUIDE][style] document.
- [x] I have included tests for the changes in my PR. If not, I have included a
  rationale for why I haven't.
- [x] My code is formatted and is accepted by `yarn fmt`.
- [x] **I understand my PR may be closed if it becomes obvious I didn't
  actually perform all of these steps.**

# Why this change is necessary and useful
In this PR I migrated the topBar component and also added redirect to the wallet component when the user starts Sheikah so you can see the top bar.

[code]: https://github.com/witnet/sheikah/blob/master/.github/CODE_OF_CONDUCT.md
[cont]: https://github.com/witnet/sheikah/blob/master/.github/CONTRIBUTING.md
[style]: https://github.com/witnet/sheikah/blob/master/docs/STYLEGUIDE.md

### CODE REVIEW

File: src/components/TopBar.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <router-link to="/wallet/transaction">Transaction</router-link>
+      </div>
+      <div class="link-tab">
+        <router-link to="/wallet/receive">Receive</router-link>
+      </div>
+      <div class="link-tab">
+        <router-link to="/wallet/send">Send</router-link>
+      </div>
+  </div>
+    <router-view></router-view>
```
Comment (Tommytrg) [inline src/components/TopBar.vue]: Fix indentation

File: src/components/TopBar.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+<script>
+export default {
+  name: 'TopBar',
+}
+</script>
+
+<style lang="scss">
+@import '@/styles/_colors.scss';
+@import '@/styles/theme.scss';
```
Comment (Tommytrg) [inline src/components/TopBar.vue]: Remove unused import

File: src/components/TopBar.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    border-bottom: 1px solid $grey-4;
+    .link-tab {
+      padding: 25px;
+    }
+    .link-tab a {
+      color: $grey-6;
+      font-weight: bold;
+      text-decoration: none;
+      padding: 20px;
+    &.router-link-active{
```
Comment (Tommytrg) [inline src/components/TopBar.vue]: Fix indentation

File: src/components/Wallet.vue [hunk 4]
```diff
@@ -1,32 +1,40 @@
...
-    </el-header>
-    <router-view></router-view>
-  </div>
+<div>
+  <TopBar />
+</div>
 </template>
 
 <script>
+import TopBar from '@/components/TopBar'
```
Comment (Tommytrg) [inline src/components/Wallet.vue]: Add blank line

File: src/components/Wallet.vue [hunk 5]
```diff
@@ -1,32 +1,40 @@
...
-<style scoped>
-.el-header {
-  font-size: 12px;
-  display: flex;
-}
-.el-header > div {
-  margin: 0 20px;
+<style lang="scss">
+@import '@/styles/_colors.scss';
+@import '@/styles/theme.scss';
```
Comment (Tommytrg) [inline src/components/Wallet.vue]: Remove unused import

File: src/components/Wallet.vue [hunk 6]
```diff
@@ -1,32 +1,40 @@
...
 </script>
 
-<style scoped>
-.el-header {
-  font-size: 12px;
-  display: flex;
-}
-.el-header > div {
-  margin: 0 20px;
+<style lang="scss">
```
Comment (Tommytrg) [inline src/components/Wallet.vue]: Should these styles be _scoped_?

**Comment by aesedepece** [comment]
- There should be no space between tabs. Replace margin with more padding. In this way, the buttons will be more easily clickable.
- Use same font color for inactive tabs as for inactive apps in the left sidebar.

**Pull request merged**